fe/fe-core/src/main/java/org/apache/doris/nereids/hint/LeadingHint.java

```diff
@@ -367,6 +435,13 @@ public Plan generateLeadingJoinPlan() {
                             newStackTop.second.getOutput(), logicalPlan.getOutput(), conditions);
                     JoinType joinType = computeJoinType(getBitmap(newStackTop.second),
                             getBitmap(logicalPlan), conditions);
+                    if (joinType == null) {
```

Comment:
better to refine this part of return type handing.

Reply:
done